dbt/adapters/events/types.py

```diff
@@ -422,3 +422,12 @@ def message(self) -> str:
             "be ignored. Set 'warn_unsupported: false' on this constraint to ignore this warning."
         )
         return line_wrap_message(warning_tag(msg))
+
+
+class TypeCodeNotFound(DebugLevel):
+    def code(self) -> str:
+        return "E050"
+
+    def message(self) -> str:
```

Comment:
I noticed on the longer messages (e.g. the two above this) we use `line_wrap_message`. Should we do that here as well? The message also appears to be longer than the linter would allow, and should probably be two or three lines.

Reply:
Thx for the suggestion! Added in e398d8b676e38c4f354ce4dd2ff5066e6f493d79